rotkehlchen/data_migrations/migrations/migration_4.py

```diff
@@ -6,6 +6,7 @@
 from rotkehlchen.constants.misc import ONE
 from rotkehlchen.fval import FVal
 from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import SupportedBlockchain
```

Comment:
Do not make a new migration. Since there is a pending DB upgrade you should just use the upgrade for this. Just add this logic in the DB upgrade.

Reply:
I had the impression that the data migration would fail if we don't add a value for the blockchain column since we finally removed the default 'ETH' value.

I admit I don't fully understand what else I must add to the DB upgrade logic. Is it ok like this?

Reply:
Hmm I think I understand. So you want to edit this migration to also add the blockchain here?

Reply:
Yeah, exactly!

Reply:
I think it makes sense then